tests/simulator/test_advance.py

```diff
@@ -74,7 +74,7 @@ def getHierarchy(
 
         extra_diag_options["mode"] = "overwrite"
         extra_diag_options["dir"] = diag_outputs
-        self.register_diag_dir_for_cleanup(diag_outputs)
+        # self.register_diag_dir_for_cleanup(diag_outputs)
```

Comment:
I was thinking maybe we could have some switch for this to keep the diags even on test pass